MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,565 @@
+    <imageEditor:SfImageEditor.Toolbars>
+        <imageEditor:ImageEditorToolbar Orientaion="Vertical"
+                                        Position="End">
+            <imageEditor:ImageEditorToolbar.ToolbarItems>
+                <imageEditor:ImageEditorToolbarItem Name="Text">
+                    <imageEditor:ImageEditorToolbarItem.SubToolbars>
+                        <imageEditor:ImageEditorToolbar>
+                            <imageEditor:ImageEditorToolbar.ToolbarItems>
+                                <imageEditor:ImageEditorToolbarItem Name="AddText" />
+                                <imageEditor:ImageEditorToolbarItem Name="TextColor" />
+                                <imageEditor:ImageEditorToolbarItem Name="Delete" />
+                            </imageEditor:ImageEditorToolbar.ToolbarItems>
+                        </imageEditor:ImageEditorToolbar>
+                    </imageEditor:ImageEditorToolbarItem.SubToolbars>
+                </imageEditor:ImageEditorToolbarItem>
+                <imageEditor:ImageEditorToolbarItem Name="Shape" />
+                <imageEditor:ImageEditorToolbarItem Name="Pen" />
+            </imageEditor:ImageEditorToolbar.ToolbarItems>
+        </imageEditor:ImageEditorToolbar>
+    </imageEditor:SfImageEditor.Toolbars>
+</imageEditor:SfImageEditor>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+imageEditor.AutoGenerateToolbarItems = false;
+ImageEditorToolbar editorToolbar = new ();
+editorToolbar.Orientaion = ToolbarOrientation.Vertical;
+editorToolbar.Position = ToolbarPosition.End;
+editorToolbar.ToolbarItems = new List<IImageEditorToolbarItem>()
+{
+    new ImageEditorToolbarItem(){ Name = "Text",
+        SubToolbars = new List<ImageEditorToolbar>()
+        {
+            new ImageEditorToolbar()
+            {
+                ToolbarItems = new List<IImageEditorToolbarItem>()
+                {
+                   new ImageEditorToolbarItem(){ Name = "AddText"},
+                   new ImageEditorToolbarItem(){ Name = "TextColor"},
+                   new ImageEditorToolbarItem(){ Name = "Delete"}
+                },
+            }
+        }
+    },
+    new ImageEditorToolbarItem(){ Name = "Shape"},
+    new ImageEditorToolbarItem(){ Name = "Pen"}
+};
+
+imageEditor.Toolbars.Add(editorToolbar);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Change toolbar item visibility
+
+Hide or show the toolbar items by specifying their icon names and setting the boolean values to true or false.
+
+N> Customize an icon by specifying its `Name`.
+
+{% tabs %}
+
+{% highlight C# %}
+
+        SfImageEditor imageEditor = new SfImageEditor();
+        imageEditor.Source = ImageSource.FromFile("image.png");
+        imageEditor.SetToolbarItemVisibility("Shape", false);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Change toolbar item enable/disable state
```

Comment:
## Change the Toolbar Item to Enable or Disable State

Reply:
Updated